.github/workflows/release.yml

```diff
@@ -95,29 +95,25 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          # We have to use this rather than the implicit GitHub token so that
+          # pushing a new tag triggers another action.
+          token: ${{ secrets.GH_TOKEN }}
       - uses: ./.github/util/initialize
         with: {github-token: "${{ github.token }}"}
 
       - name: Deploy
         run: dart run grinder protobuf pkg-pub-deploy
         env: {PUB_CREDENTIALS: "${{ secrets.PUB_CREDENTIALS }}"}
 
-  deploy_sass_api:
-    name: Deploy sass_api
-    runs-on: ubuntu-latest
-    needs: [deploy_pub]
-
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/util/initialize
-        with: {github-token: "${{ github.token }}"}
-
-      - name: Deploy
-        run: dart run grinder deploy-sass-api
-        env:
-          PUB_CREDENTIALS: "${{ secrets.PUB_CREDENTIALS }}"
-          GH_TOKEN: "${{ secrets.GH_TOKEN }}"
-          GH_USER: sassbot
+      - name: Get Sass API version
+        id: sass-api-version
+        run: |
+          echo "version=$(cat pkg/sass_api/pubspec.yaml | sed -nE 's/version: (.*)/\1/p')" | tee --append "$GITHUB_OUTPUT"
+      # This should be /-separated rather than hyphenated, but pub.dev doesn't
+      # currently allow that (dart-lang/pub-dev#8690).
+      - run: git tag sass-api-${{ steps.sass-api-version.outputs.version }}
```

Comment:
Not entirely sure, but another way this might be broken is that `git` might complain that "user" is not setup. Maybe we need to add these before creating the tag:
```
git config user.email 
git config user.name 
```

Reply:
I believe lightweight tags don't include any author information. (Note that the git user isn't set when tagging `sass-parser` below.)